docs/.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = F405,F403,F841,E722 
```

Comment:
?

Reply:
F403 and F405 are to do with import * which we aren't currently fixing.
F841 is unused variables this is mostly comes up next to commented out code that used the variable someone will need to check why this is commented out / unused in future.
E722 is using bare except statements someone needs to go look at these in the future and work out what there supposed to be catching

I've updated #50 with notes to fix these

Reply:
cool